apps/gallery/js/startup.js

```diff
@@ -280,6 +280,7 @@
           function() {
             var headers = document.querySelectorAll('gaia-header');
             for (var i = 0; i < headers.length; ++i) {
+              headers[i].firstElementChild.removeAttribute('hidden');
```

Comment:
This is a little brittle since it assumes that hte <h1> will always be the first child of <gaia-header>. That's probably true, but it still might be better to do 'headers[i].querySelectorAll('h1').forEach(h => h.hidden = false);` or something like that.